### PR TITLE
Some improvements to the makefile.

### PR DESCRIPTION
**Pull request by matiasdelellis**
* Build is the new default rule.
 * Add a rule 'test-bin'deps' that checks the dependencies to build the application.
 * Separate the rules for translations, not needed unless you want to add a translation.
 * Move js deps and models to vendor folder.

### CODE REVIEW

**Comment by matiasdelellis** [comment]
Branko..
Can you install handlebars on travis ??

File: Makefile [hunk 1]
```diff
@@ -95,13 +125,25 @@ appstore:
...
 
 clean:
 	rm -rf ./build
-	rm -f js/handlebars.js js/lozad.js
-	rm -f models/1/mmod_human_face_detector.dat
-	rm -f models/1/dlib_face_recognition_resnet_model_v1.dat
-	rm -f models/1/shape_predictor_5_face_landmarks.dat
-	rm -r translationtool.phar
\ No newline at end of file
+	rm -f vendor/autoload.php
```
Comment (stalker314314) [inline Makefile]: can this be simplified with deleting everything from vendor, including it? Scary `rm -rf vendor`?:)

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +23,9 @@ env:
     - DB=sqlite CORE_BRANCH=stable15
 
 before_install:
+  - sudo apt-get install npm
+  - sudo npm config set strict-ssl false
```
Comment (stalker314314) [inline .travis.yml]: cool that it works:D

File: Makefile (hunk 1 above)
Comment (matiasdelellis) [reply Makefile]: We should not delete the files LICENCE.md and README.md of the models...

**Pull request merged**